projboard/tests/test_client.py

```diff
@@ -5,3 +5,19 @@
 def test_homepage(client):
```

Comment:
Please add template check here

Reply:
0596f9817cd66413350d096381a26dd528fb5608 You mean that check?